index.bs

```diff
@@ -3457,6 +3494,29 @@ To <dfn noexport>set an OS-support header</dfn> given a [=header list=]
 1. [=header list/Set a structured field value=] given
     ("<code>[=Attribution-Reporting-Support=]</code>", |dict|) in |headers|.
 
+To <dfn>obtain and deliver debug reports on OS registrations</dfn> given an
+[=OS debug data type=] |dataType|, a [=list=] of [=OS registrations=] |registrations|,
+and an [=origin=] |contextOrigin|:
+
+1. If |registrations| [=list/is empty=], return.
+1. Let |contextSite| be the result of [=obtaining a site=] from |contextOrigin|.
+1. [=list/iterate|For each=] |registration| of |registrations|:
+    1. If |registration|'s [=OS registration/debug reporting enabled=] is false, [=iteration/continue=].
+    1. Let |origin| be |registration|'s [=OS registration/URL=]'s [=url/origin=].
+    1. If |origin| is not [=check if an origin is suitable|suitable=], [=iteration/continue=].
+    1. Let |body| be a new [=map=] with the following key/value pairs:
+        : "`context_site`"
```

Comment:
I don't feel strongly, but for consistency, just pointing out that for web-based debug reports, we use the terms `attribution_destination` and `source_site`. It appears that in this case we aren't saying whether the registration was for a source or a trigger. Arguably the reporting origin can determine that from the registration URL, but it's possible for a single registration URL to be used for both types.

Reply:
Never mind; this is indicated in the type.